docs/manuals/core/concepts/sequence.md

```diff
@@ -0,0 +1,32 @@
+A `Sequence^` is designed to model an algorithm. It represents a direct acyclic graph of input, intermediate, and output
+data nodes linked together by tasks. A _sequence_ is a set of tasks designed to perform a set of functions.
+
+For instance, in a typical machine learning application, we may have several sequences: a _sequence_ dedicated to
+preprocessing and preparing data, a _sequence_ for computing a training model, and a _sequence_ dedicated to scoring.
+
+!!! example "In the example"
+
+    We have chosen to model only two sequences corresponding to a manufacturer having to first predict the sales
+    forecast, then based on the sales forecast to plan its production in its plant.
```

Comment:
"forecast, then based on the sales forecast to plan" -> "forecast then, based on the sales forecast, plan"